src/App/Document.cpp

```diff
@@ -2986,19 +2986,17 @@ void Document::getLinksTo(std::set<DocumentObject*> &links,
 {
     std::map<const App::DocumentObject*, std::vector<App::DocumentObject*> > linkMap;
 
-    for(auto o : objs.size()?objs:d->objectArray) {
-        if(o == obj) continue;
+    for(auto o : !objs.empty()?objs:d->objectArray) {
+        if (o == obj) continue;
         auto linked = o;
-        if(options & GetLinkArrayElement)
-            linked = o->getLinkedObject(false);
-        else {
+        if (options & GetLinkArrayElement) {
+            linked = o->getLinkedObject(true, nullptr, Prop_Output | Prop_Hidden, 0);

```

Comment:
This changes the semantic. Before the change the first parameter was false and now it's true.

Reply:
Oops. I tried to fix not having defaults in virtual, and messed up, then obviously messed up further trying to restore. Will restore.